docs/orchestrating-elastic-stack-applications/elasticsearch/snapshots.asciidoc

```diff
@@ -10,72 +10,35 @@ endif::[]
 
 To set up automated snapshots for Elasticsearch on Kubernetes you have to:
 
-. Ensure you have the necessary Elasticsearch storage plugin installed.
-. Add snapshot repository credentials to the Elasticsearch keystore.
 . Register the snapshot repository with the Elasticsearch API.
-. Set up a https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/[CronJob] to take snapshots on a schedule.
+. Set up a Snapshot Lifecycle Management Policy via https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-lifecycle-management-api.html[API] or the https://www.elastic.co/guide/en/kibana/current/snapshot-repositories.html[Kibana UI]
```

Comment:
```suggestion
. Set up a Snapshot Lifecycle Management Policy through https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-lifecycle-management-api.html[API] or the https://www.elastic.co/guide/en/kibana/current/snapshot-repositories.html[Kibana UI]
```
Replace latinisms with more accessible terms. Check our [Accessibility Tips and tricks](https://docs.google.com/document/d/1IGuMf16-cwxx8BX1Z9PWsFWsTPxicwTkX_r7UOhjd4E/edit)